archive/terraform/main.tf

```diff
@@ -2,7 +2,7 @@
 
 resource "aws_api_gateway_base_path_mapping" "api-gw-mapping-241118" {
   api_id      = "${module.stack-241118.api_gateway_id}"
-  domain_name = "api-stage.wellcomecollection.org"
+  domain_name = "stage.wellcomecollection.org"
   base_path   = "storage"
```

Comment:
I think I preferred `api-stage` & `api` rather than `stage` & `api`, but I expect there is a reason this is now different.